### PR TITLE
add  the usage of  configuration  "spark.sql.oap.index.directory" in README

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?

Add  the usage of  configuration  "spark.sql.oap.index.directory" in README

## How was this patch tested?

no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,10 @@ Compression Codec - Choose compression type for OAP data files.
 * Usage1: `sqlContext.conf.setConfString(SQLConf.OAP_COMPRESSION.key, "SNAPPY")`
 * Usage2: `CREATE TABLE t USING oap OPTIONS ('compression' 'SNAPPY')`
 
+Index Directory Setting - Enable OAP support to separate the index file in specific directory. The index file is in the directory of data file in default.
+* Default: ""
+* Usage: `sqlContext.conf.setConfString(SQLConf.OAP_INDEX_DIRECTORY.key, "")`
```
Comment (gczsjdy) [inline README.md]: Use DDL for this? More people use by SQL interface.
And maybe it's better to place this as the 2nd item in `Configurations and Performance Tuning`, then the first 2 are configurations for functionalities, others are configurations for performance tuning.

Comment (JkSelf) [reply README.md]: @gczsjdy updated.

File: README.md [hunk 2]
```diff
@@ -70,6 +70,11 @@ Parquet Support - Enable OAP support for parquet files
 * Default: true
 * Usage: `sqlContext.conf.setConfString(SQLConf.OAP_PARQUET_ENABLED.key, "false")`
 
+Index Directory Setting - Enable OAP support to separate the index file in specific directory. The index file is in the directory of data file in default.
+* Default: ""
+* Usage1: `sqlContext.conf.setConfString(SQLConf.OAP_INDEX_DIRECTORY.key, "/tmp")`
+* Usage2: `SET spark.sql.oap.index.directory = "/tmp"`
```
Comment (gczsjdy) [inline README.md]: Should be `= tmp`, I tried the one with quotation and it doesn't work...

Comment (JkSelf) [reply README.md]: @gczsjdy ,yes, updated.

**Comment by gczsjdy** [comment]
Thanks @JkSelf @LuciferYang !

**Pull request merged**